src/components/Movies.jsx

```diff
@@ -0,0 +1,22 @@
+import Movie from './Movie'
+import '../styles/movies.scss'
+
+const Movies = ({ movies, viewTrailer, closeCard }) => {
```

Comment:
There's an unused parameter, closeCard, in the component.

Keeping our code streamlined is always beneficial, and avoiding unused parameters across the project helps maintain clarity and simplicity.